sdk/iothub/src/service/responses/device_twin_response.rs

```diff
@@ -54,6 +54,8 @@ impl std::convert::TryFrom<Response<bytes::Bytes>> for DeviceTwinResponse {
     fn try_from(response: Response<bytes::Bytes>) -> Result<Self, Self::Error> {
         let body = response.body();
 
+        debug!("body == {:#?}", std::str::from_utf8(body));
```

Comment:
I'd like to not use `debug!` for this. I'm aware we're using this in other parts of the SDK but those should also be removed. If we feel this is necessary logging `trace` feels much more appropriate. Even still, a bunch of logs with just "body = $RANDOM_JSON" does not feel particularly useful to me.